README.md

```diff
@@ -24,12 +24,13 @@ gem 'apimatic_core_interfaces'
 ```
 
 ## Interfaces
-| Name                                                                                 | Description                                                                                |
-|--------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
-| [`HttpClient`](lib/apimatic-core-interfaces/client/http_client.rb)                   | To save both Request and Response after the completion of response                         |
-| [`ResponseFactory`](lib/apimatic-core-interfaces/factories/response_factory.rb)      | To convert the client-adapter response into a custom HTTP response                         |
-| [`Authentication`](lib/apimatic-core-interfaces/types/authentication.rb)             | To setup methods for the validation and application of the required authentication scheme  |
-| [`ClientConfiguration`](lib/apimatic-core-interfaces/client/client_configuration.rb) | To setup the http client configurations including retries, timeouts and connections etc.  |
+| Name                                                                                 | Description                                                                               |
+|--------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
+| [`HttpClient`](lib/apimatic-core-interfaces/client/http_client.rb)                   | To save both Request and Response after the completion of response                        |
+| [`ResponseFactory`](lib/apimatic-core-interfaces/factories/response_factory.rb)      | To convert the client-adapter response into a custom HTTP response                        |
+| [`Authentication`](lib/apimatic-core-interfaces/types/authentication.rb)             | To setup methods for the validation and application of the required authentication scheme |
+| [`UnionType`](lib/apimatic-core-interfaces/types/union_type.rb)                      | To setup methods for the validation and deserialization of OneOf/AnyOf union types   |
```

Comment:
Blocker: Serialization section needs to be included in the description since we are now handling the serialization as well.

Reply:
Done